apps/folder/pkg/apis/folder/v1beta1/types.go

```diff
@@ -90,3 +69,6 @@ func UnstructuredToDescendantCounts(obj *unstructured.Unstructured) (*Descendant
 	err := runtime.DefaultUnstructuredConverter.FromUnstructured(obj.Object, &res)
 	return &res, err
 }
+
+// Empty stub
+type FolderStatus struct{}
```

Comment:
Once @IfSentient has a fix for this in the SDK, we should be able to get rid of this!